components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -154,6 +158,23 @@ - (void)viewWillLayoutSubviews {
   self.tableView.contentOffset = CGPointMake(0, -size.height);
 }
 
+- (CGFloat)openingSheetHeight {
+  // If there are too many options to fit on half of the screen then show as many options as
+  // possible minus half a cell, to allow for bleeding and signal to the user that the sheet is
+  // scrollable content.
+  CGFloat maxHeight = CGRectGetHeight(self.view.bounds) / 2;
+  CGFloat headerHeight = [self.header sizeThatFits:CGRectStandardize(self.view.bounds).size].height;
+  CGFloat cellHeight = self.tableView.contentSize.height / (CGFloat)_actions.count;
+  NSInteger amountOfCellsToShow = (NSInteger)(maxHeight - headerHeight) / (NSInteger)cellHeight;
```

Comment:
nit: any reason why not to round after calculation in done (since all values are floats)?
( NSInteger ) ( ( maxHeight - headerHeight ) / cellHeight )

Reply:
Done